pandas/_libs/src/klib/khash_python.h

```diff
@@ -13,25 +13,31 @@
 // is 64 bits the truncation causes collission issues.  Given all that, we use our own
 // simple hash, viewing the double bytes as an int64 and using khash's default
 // hash for 64 bit integers.
-// GH 13436
+// GH 13436 showed that _Py_HashDouble doesn't work well with khash
```

Comment:
can you add a link here (or in khash.h) to this PR and some text to indicate the hashing strategy so its easy to understand from reading this code.

Reply:
I hope this is enough: https://github.com/pandas-dev/pandas/pull/36729/commits/f2496381d1dd088df60f29e6a9604dce30abaf1f#diff-7ac30c345bd6d38838a46337e4c6b5b6feae3e1fd5aea54b5d9e37d20054edf5R18